src/interaction/joystickcamerastates.cpp

```diff
@@ -67,49 +72,55 @@ void JoystickCameraStates::updateStateFromInput(const InputState& inputState,
             value *= -1.f;
         }
 
-        value = static_cast<float>(value * _sensitivity);
+        if (std::fabs(t.sensitivity) > std::numeric_limits<double>().epsilon()) {
```

Comment:
`std::abs` should be fine here since `t.sensitivity` is already a `float`?  And should probably be compared to `std::numeric_limits<float>::epsilon()`   Checking against `double` would be the same as checking against > as the double-epsilon will be smaller than the smaller representable float.   and b) no need to create an object of numeric_limits and then call `epsilon()` on that object since `epsilon` is a static function

Reply:
Both `t.sensitivity` and `_sensitivity` are doubles, so then it should still be `std::fabs` and `std::numeric_limits<double>().epsilon()` right?

Reply:
Ah, yes to `double` in numeric limits, but you can still use `std::abs`  (see (2) in https://en.cppreference.com/w/cpp/numeric/math/fabs).  No need to worry about the type of the value then.

And `std::numeric_limits<double>::epsilon()`, not `std::numeric_limits<double>().epsilon()` :)